### PR TITLE
User Model for Phase 1

### PR DESCRIPTION
**Pull request by mmadanire**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: server/db/models/user.js [hunk 1]
```diff
@@ -24,10 +27,33 @@ const User = db.define('user', {
       return () => this.getDataValue('salt')
     }
   },
-  googleId: {
-    type: Sequelize.STRING
-  }
-})
+  isAdmin: {
+    type: Boolean,
```
Comment (hpr) [inline server/db/models/user.js]: must be sequelize.boolean

Comment (mmadanire) [reply server/db/models/user.js]: Added 'Sequelize'

Comment (hpr) [reply server/db/models/user.js]: thank you Mercedes this looks excellent

Comment (hpr) [reply server/db/models/user.js]: actually it needs to be capitalized

Comment (mmadanire) [reply server/db/models/user.js]: Thanks for your feedback. Boolean is not in caps. Plus the seed file is now updated with additional model info.

File: script/seed.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ async function seed() {
   console.log('db synced!')
 
   const users = await Promise.all([
-    User.create({email: 'cody@email.com', password: '123'}),
-    User.create({email: 'murphy@email.com', password: '123'})
+    User.create({email: 'cody@email.com', password: '123', isAdmin: 'false', firstName: 'Cody', lastName: 'Barnes'}),
+    User.create({email: 'murphy@email.com', password: '123', isAdmin: 'true', firstName: 'Murphy', lastName: 'Clark'})
```
Comment (hpr) [inline script/seed.js]: bool values can't be in quotes

File: script/seed.js [hunk 3]
```diff
@@ -8,8 +8,8 @@ async function seed() {
   console.log('db synced!')
 
   const users = await Promise.all([
-    User.create({email: 'cody@email.com', password: '123'}),
-    User.create({email: 'murphy@email.com', password: '123'})
+    User.create({email: 'cody@email.com', password: '123', isAdmin: 'false', firstName: 'Cody', lastName: 'Barnes'}),
```
Comment (hpr) [inline script/seed.js]: bool values can't be in quotes

Comment (mmadanire) [reply script/seed.js]: Database successfully seeded

**Pull request merged**